packages/blocks/src/limitHelpers.ts

```diff
@@ -0,0 +1,233 @@
+import R, {
+  concat,
+  curry,
+  flip,
+  has,
+  ifElse,
+  lensIndex,
+  lensProp,
+  lt,
+  map,
+  mapObjIndexed,
+  mergeLeft,
+  of,
+  over,
+  pick,
+  pipe,
+  prop,
+  take,
+  values,
+  view,
+} from 'ramda';
+
+const dot3 = '...';
```

Comment:
```suggestion
const dot3 = '…';
```

Didn't want to use the ellipse character? 😛

Reply:
〝Smart ellipsis〞?